### PR TITLE
Upgrade to Symfony 3.0

### PR DESCRIPTION
**Pull request by j0k3r**
Bundle which needs to be updated first:

- [x] [SchebTwoFactorBundle](https://github.com/scheb/two-factor-bundle)
- [x] [LexikFormFilterBundle](https://github.com/lexik/LexikFormFilterBundle)
- [x] [FOSOAuthServerBundle](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle)
- [x] [FOSUserBundle](https://github.com/FriendsOfSymfony/FOSUserBundle)
- [x] [RulerZBundle](https://github.com/K-Phoen/RulerZBundle)

### CODE REVIEW

**Comment by nicosomb** [comment]
Do you plan to fix these bundles? Or do you think we have to wait for updates.

**Comment by j0k3r** [comment]
I've started to fix the first one.
Other bundles have PRs for that, we just need to wait for them to be reviewed and merged.

**Comment by nicosomb** [comment]
Yeah! Still in WIP?

File: .gitignore [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
 /app/config/parameters.yml
-/app/config/parameters.ini
 
 # Managed by Composer
-/app/bootstrap.php.cache
-/var/bootstrap.php.cache
-/bin/*
-!bin/console
-!bin/symfony_requirements
 /vendor/
```
Comment (nicosomb) [inline .gitignore]: These lines are missing: 

```
# Docker container logs and data
docker/logs/
docker/data/
```

File: bin/console [hunk 2]
```diff
@@ -1,22 +1,24 @@
 #!/usr/bin/env php
```
Comment (nicosomb) [inline bin/console]: Are the permissions correct on this file?

**Comment by nicosomb** [comment]
In `README.md`, you have to change with that:

```
php bin/console wallabag:install
php bin/console server:run
```

File: .gitignore (hunk 1 above)
Comment (j0k3r) [inline .gitignore]: Yes, this is because my branch is outdated regarding the master.

Comment (nicosomb) [inline .gitignore]: v2, not master ;-)

Comment (j0k3r) [inline .gitignore]: You get it :grin:

File: bin/console (hunk 2 above)
Comment (j0k3r) [inline bin/console]: Good catch, :arrow_right: `755`

**Comment by j0k3r** [comment]
:tada: :boom:

**Comment by nicosomb** [comment]
:beers:

**Pull request merged**